test/Tester/StreamingTests/PullingAgentManagementTests.cs

```diff
@@ -12,18 +13,15 @@
 
 namespace UnitTests.StreamingTests
 {
-    [DeploymentItem("OrleansConfigurationForStreamingUnitTests.xml")]
-    [DeploymentItem("ClientConfigurationForStreamTesting.xml")]
-    [DeploymentItem("OrleansProviders.dll")]
-    [TestClass]
-    public class PullingAgentManagementTests : HostedTestClusterPerFixture
+    public class PullingAgentManagementTests : OrleansTestingBase, IDisposable
     {
         private const string adapterName = StreamTestsConstants.AZURE_QUEUE_STREAM_PROVIDER_NAME;
         private readonly string adapterType = typeof(AzureQueueStreamProvider).FullName;
+        private readonly TestingSiloHost HostedCluster;
 
-        public static TestingSiloHost CreateSiloHost()
+        public PullingAgentManagementTests()
```

Comment:
this used to be HostedTestClusterPerFixture, but now it's restarting per test. I'm fixing it now